core/modules/installer/css/installer.css

```diff
@@ -0,0 +1,251 @@
+#installer-browser-install-button-form {
+  clear: both;
+  margin: 10px 0 0;
+}
+
+#installer-browser-install-queue {
+  margin: 5px 0 0;
+}
+
+.installer-browser-install-queue-item {
+  margin: 5px 0;
+}
+
+.installer-browser-install-queue-items {
+  clear: both;
+  margin: 0 0 15px;
+}
+
+.installer-browser-install-link {
+  clear: both;
+}
+
+.installer-browser-install-queue-item a {
+  padding-left: 20px;
+  background: url("../images/red-x.png") no-repeat scroll left 2px transparent;
+}
+
+.installer-browser-selected-release {
+  display: none;
+}
+
+.installer-browser-show-releases-link {
+  cursor: pointer;
+}
+
+div.item-list ul.installer-browser-sort-list, 
+div.item-list ul.installer-browser-servers-list {
+  margin: 0;
+}
+
+div.item-list ul.installer-browser-sort-list li, 
+div.item-list ul.installer-browser-servers-list li {
+  display: inline;
+  list-style-image: none;
+  margin-right: 10px;
+  float: left;
+  clear: both;
+}}
+
+div.item-list ul.installer-browser-sort-list li a, 
+div.item-list ul.installer-browser-servers-list li a {
+  color: #0074BD;
+}
+
+div.item-list ul.installer-browser-sort-list li.sort-active a,
+div.item-list ul.installer-browser-servers-list li.server-active a {
+  color: #000;
+}
+
+div.item-list ul.installer-browser-sort-list li.sort-header,
+div.item-list ul.installer-browser-servers-list li.server-header {
+  font-weight: bold;
+}
+
+div.item-list ul.installer-browser-sort-list li.sort-asc {
+  background: url("../images/arrow-asc.png") no-repeat scroll right 2px transparent;
+  padding-right: 15px;
+}
+
+div.item-list ul.installer-browser-sort-list li.sort-desc {
+  background: url("../images/arrow-desc.png") no-repeat scroll right 2px transparent;
+  padding-right: 15px;
+}
```

Comment:
Instead of new arrows, seems like we could re-use `theme('tablesort_indicator')` to use core's arrow-asc/desc.png images.